ui/conftest.py

```diff
@@ -41,6 +41,23 @@ def video(user):  # pylint: disable=redefined-outer-name
 
```

Comment:
At the top of this file there is an import from `settings` like `from odl_video import settings`. Can you change that to `from django.conf import settings`?